### PR TITLE
Training update

### LINKED ISSUE #589: Improve Training Page
- [x] Highlight training courses that are currently scheduled or are likely to be scheduled.
- [x] Make courses sound more appealing.
- [x] Review organisation of content.

### PR DESCRIPTION
**Pull request by bobturneruk**
Resolves #589

I think this is an improvement - I think the lists of resources could do with being reviewed.

### CODE REVIEW

File: _data/event-categories.yml [hunk 1]
```diff
@@ -4,6 +4,9 @@ seminar:
 dltraining:
     image: "/assets/images/icons/icons8-connect-50.png"
     text: "Workshop"
+gitzerohero:
```
Comment (bobturneruk) [inline _data/event-categories.yml:7]: This lets us filter git courses out of events.

File: _data/navigation.yml [hunk 2]
```diff
@@ -18,16 +18,14 @@
       url: /collaboration/testimonials
 - title: "Training"
   subitems:
```
Comment (ns-rse) [inline _data/navigation.yml:20]: Why has the training statement been removed from the navigation?

File: _data/event-categories.yml (hunk 1 above)
Comment (ns-rse) [reply _data/event-categories.yml:7]: Makes sense to move these out to their own category. :+1:

File: pages/training/index.md [hunk 3]
```diff
@@ -6,36 +6,44 @@ type: text
 permalink: /training/
 ---
 
-We sometimes run courses for free (see our [events page][events]). However, we usually need to seek funding from an organisation, department, research group, institute or doctoral training centre. The costs of running a course are our [staff time][service] plus venue and infrastructure costs. This allows us to customise or develop new training for specific audiences.  
+People working code for research, whether they are writing their first script to managing a complex software project need skills that academic training might not provide. The RSE Team's aspiration is that all researchers who need these skills should be given the opportunity to learn them, either from the RSE Team, our University of Sheffield colleagues or other sources.
```
Comment (ns-rse) [inline pages/training/index.md]:
```suggestion
People working with code for research, whether they are writing their first script to managing a complex software project need skills that academic training might not provide. The RSE Team's aspiration is that all researchers who need these skills should be given the opportunity to learn them, either from the RSE Team, our University of Sheffield colleagues or other sources.
```

File: pages/training/index.md [hunk 4]
```diff
@@ -6,36 +6,44 @@ type: text
...
 - [Contributing to Open Source Software](courses/Open_Source.md)
-- [git & GitHub through GitKraken - from Zero to Hero!](courses/git_Hero.md)
 - [Write better research software (in Python)](courses/good_soft.md)
 
-If you're interested in us running similar courses for your learners, please contact <rse@sheffield.ac.uk>.
-
 Our expertise for bespoke training development includes Python, R, MATLAB, Reproducible research, C and C++, git and GitHub, Linux shell, High Performance Computing, GPU Computing, Systems administration inc. Docker and Ansible.
 
-### Carpentries
+We can run courses where material is made openly available by others (e.g. The Carpentries).
```
Comment (ns-rse) [inline pages/training/index.md]: Readers may not be aware of what The Carpentries are (although I see now I've scrolled down they are linked from the bottom, links in-line are easy to include, disregard at your discretion)...

```suggestion
We can run courses where material is made openly available by others (e.g. [The Carpentries](https://carpentries.org)).
```

**Review by ns-rse — COMMENTED** [review]
All great improvements, some very minor points spotted.

File: _data/navigation.yml (hunk 2 above)
Comment (bobturneruk) [reply _data/navigation.yml:20]: I wanted to make sure it was tied closely to the training courses stuff, and reduce the number of links people have to read.

File: _data/navigation.yml [hunk 5]
```diff
@@ -18,16 +18,14 @@
...
-      url: /training/statement
     - title: Training Courses
       url: /training/
     - title: Researcher Programming Resources
       url: /training/programming
+    - title: Researcher Deep Learning Resources
+      url: /training/deeplearning
     - title: Code Clinic
       url: /support/code-clinic
-    - title: Deep Learning
```
Comment (mondus) [inline _data/navigation.yml:29]: I wonder if this (and the link title page and intro) should perhaps be changed to "Artificial Intelligence and Machine Learning" to reflect the universities interests. Likewise the page could do with an update to provide some links back to the AI network and any other institutional resources.

File: pages/training/deeplearning/index.md [hunk 6]
```diff
@@ -5,14 +5,6 @@ slug: index
 type: text
 ---
 
-In conjunction with [Nvidia Deep Learning Institue (DLI)](https://www.nvidia.com/dli), RSES offers various practical workshops on how to apply deep learning to your research. Details for these events will be posted on this page and shared via the [RSE announcement mailing list](/community/).
```
Comment (mondus) [inline pages/training/deeplearning/index.md:8]: Could do with some general blurb and links to local resources and networks. I know this is a training page but it seems the most appropriate place to mention the AI network. 

If your making edits I also think he list of resources would be better above the previous events.

**Review by mondus — CHANGES_REQUESTED** [review]
Looks good. A change relating to terminology around AI/ML suggested.

File: _data/navigation.yml (hunk 5 above)
Comment (bobturneruk) [reply _data/navigation.yml:29]: I've changed the title.

File: pages/training/deeplearning/index.md (hunk 6 above)
Comment (bobturneruk) [reply pages/training/deeplearning/index.md:8]: Previous events should have gone prior to you seeing this. Hopefully just a rendering / caching thing.

Comment (bobturneruk) [reply pages/training/deeplearning/index.md:8]: I've put a mention in. Both the programming and AI/ML pages could do with separate reviews.

File: _data/navigation.yml (hunk 5 above)
Comment (bobturneruk) [reply _data/navigation.yml:29]: Some links in. I've been pretty minimal here. Let me know if more is needed.

**Comment by bobturneruk** [comment]
I've snuk in some pretty brittle looking jekyll to help people sign up to the mailing list if there are no current courses.

File: _data/navigation.yml (hunk 5 above)
Comment (mondus) [reply _data/navigation.yml:29]: Looks good

File: pages/training/deeplearning/index.md (hunk 6 above)
Comment (mondus) [reply pages/training/deeplearning/index.md:8]: Can always expand upon this later.

**Review by mondus — APPROVED** [review]

**Pull request merged**